### ISSUE TITLE
New SteamOS Update - Plugin Entry Doesn't Display

### ISSUE THREAD

**Issue by CartCollector** [WerWolv/SDH-SystemToolbox#8]
A new update went out and the plugin generates errors again when loading.  I assume it needs to be updated?  I really enjoy the SSH server aspect of this!

**Comment by GMRobo** [comment]
> I really enjoy the SSH server aspect of this!

Yeah, currently I use Decky Terminal and it is not as elegant as this was. +1 on updating this

**Comment by wian007** [comment]
Is this fix already released?

**Comment by GMRobo** [comment]
> Is this fix already released?

As far as I know, nothing on stable branch thus far
Nothing on testing either

**Comment by patrickkempff** [comment]
is this project dead?
